app/js/controllers/main.js

```diff
@@ -70,6 +78,23 @@ export default class MainController extends BaseController {
   }
 
   /**
+   * Clear all data/settings stored on the browser. Use with caution.
+   *
+   * @param {boolean} ignoreServiceWorker
+   * @return {Promise}
+   */
+  clear(ignoreServiceWorker = true) {
+    const promises = [this[p.settings].clear()];
+
+    if (!navigator.serviceWorker && !ignoreServiceWorker) {
+      promises.push(navigator.serviceWorker.ready
+        .then((registration) => registration.unregister()));
+    }
+
+    return Promise.all(promises);
+  }
```

Comment:
What's the benefit of moving this out of `server/index.js`?
I'd like to get rid of the controllers in favour of `react-router` in a near future, so let's not add extraneous methods to it.
Also the responsibility of anything related to service workers is on the `server` lib.